### DISCUSSION TITLE
GITHUB ACCOUNT SUSPENDED FOR NO REASON

### DISCUSSION THREAD

**Discussion by Shubham0041** [community/community#131956] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub Community,

I'm writing to seek assistance regarding my GitHub account, which has been suspended for over 4 weeks without any explanation. This unexpected suspension occurred without prior notice, and despite my attempts to inquire about the reason, I have not received any response from GitHub support. As someone who has always endeavored to comply with GitHub's terms of service and community guidelines, this situation has left me confused and concerned. The prolonged suspension is significantly impacting my work and collaborations, and I'm at a loss for how to proceed.

I'm reaching out to this community in hopes that someone might have experienced a similar situation or can provide guidance on how to effectively escalate this matter and get in touch with the appropriate GitHub support team. Any advice or assistance you can offer would be immensely appreciated as I try to resolve this issue and regain access to my account. For reference, my ticket number is Ticket ID: 2849590, my GitHub username is Shubham6621, and my email address is [shubhamvmali06@gmail.com] Thank you for any help or advice you can provide.

Ticket ID: 2849590
GITHUB USERNAME - Shubham6621
Gmail - shubhamvmali06@gmail.com

**Comment by paiineli** [comment]
Dear Shubham6621,

I’m sorry to hear about the difficulties you're facing with your GitHub account suspension. It’s understandably frustrating to be in this situation, especially when you have not received any communication regarding the reason for the suspension.

Here are a few steps you can take to try to resolve this issue:

Follow Up on Your Ticket: If you haven’t already, consider following up on your existing support ticket (Ticket ID: 2849590). A polite inquiry about the status of your ticket may prompt a response from GitHub support.

Check Email Filters: Ensure that any communications from GitHub are not being filtered into your spam or junk folder. Sometimes responses can get lost in email filters.

Use Social Media: You might try reaching out to GitHub’s support team on their official Twitter account (@GitHubHelp). Sometimes social media can expedite responses.

Community Forums: While you’ve reached out here, continuing to engage with community forums may help. Other users may have similar experiences and could provide insight or support.

Review GitHub Policies: Make sure to review GitHub’s terms of service and community guidelines to ensure there weren’t any inadvertent violations.

I hope you receive a response soon and can regain access to your account. Best of luck!

Best regards,

**Comment by LiteBrite82** [comment]
Hi @Shubham0041 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
